nav/stepper/motion.py

```diff
@@ -18,8 +17,35 @@ def __init__(self, GPIO):
 		[0.0.0] Rishav
 			--- Created function
 	"""
-	def mov(self, dir, distance):
-		print(dir, distance)
+	def mov(self, dir, distance, GPIO):
+            for pin in self.step_mode_select:
+                GPIO.output(pin, GPIO.HIGH)
+                
+            distance = int(distance* STEPS_PER_INCH)
+            sleep(1)
+                
+            if(dir == 0):
```

Comment:
This is not a good way to do this. If dir is not 0 or 1, then what? Need to have proper error handling and logging.